packages/solid-table/src/createTable.ts

```diff
@@ -38,6 +41,7 @@ export function createTable<
   createComputed(() => {
     table.setOptions((prev) => {
       return mergeProps(prev, tableOptions, {
+        _features: { ...coreFeatures, ...tableOptions._features },
```

Comment:
Not sure about this, without that I think the new options will not contains the coreFeatures